### ISSUE TITLE
Some Results on MVTec bottle

### ISSUE THREAD

**Issue by alexriedel1** [rximg/EfficientAD#4]
Hi, here are some qualitative and quantitative results on MVTec bottle defect of the EfficientAD(M) method.
```
image_AUROC: 1.0
image_F1Score: 1.0
pixel_AUROC: 0.9876494407653809
pixel_F1Score: 0.7927650213241577
```
Patchcore on Wide ResNet-50 achieves image_AUROC: 1.0 and pixel_AUROC: 0.984

![000](https://user-images.githubusercontent.com/54716527/235113149-1c33a160-4da0-4a48-8586-0e34e033fc63.png)
![014](https://user-images.githubusercontent.com/54716527/235113227-a88648f9-804a-4b53-aef5-169846661526.png)
![015](https://user-images.githubusercontent.com/54716527/235113302-2ef6b2c3-4abd-4e3a-9ce4-f6accead5f26.png)

**Comment by rximg** [comment]
This is an exciting news, indicating that our reproduction is successful. I plan to run a set of MVTec results and add them to the readme. However, the datasets mentioned in the paper are VisA and MVTec LOCO, which I haven't used much. It may take some time to include them. If you are going to integrate the code for dataset loading and testing these two datasets, you can submit a PR after it's completed, or you can wait for me to add them in about two weeks.

**Comment by alexriedel1** [comment]
Yess I'm very optimistic in the implementation :) 
I'm working on Integrating your work in https://github.com/openvinotoolkit/anomalib so benchmarks will be very easy to make then

**Comment by rximg** [comment]
Okay, it's my pleasure

**Comment by nelson1425** [comment]
@alexriedel1 You can also use this as a reference implementation. It has the same results as in the paper: https://github.com/nelson1425/EfficientAD

**Comment by rximg** [comment]
@alexriedel1 When the batch_size > 1 during training, incorrect results are obtained. Have you encountered this issue during testing?

**Comment by alexriedel1** [comment]
@nelson1425 @rximg I have created a pull request here: https://github.com/openvinotoolkit/anomalib/pull/1073
You can make yourself familiar with anomalib and work on this together :) 

The last thing thats not working is an issue in the feature map normalization. Normalizing the feature maps with the calculated quantiles somehow breaks the model results and I don't know why


@rximg batch size 1 is intended by the paper authors too. How to use larger batch sizes needs to be looked into I guess

**Comment by samet-akcay** [comment]
@rximg, @nelson1425 and @alexriedel1, I appreciate your contribution. This is amazing!

**Comment by nelson1425** [comment]
@alexriedel1 Thank you for anomalib, I will comment things which I noticed during reproducing results.

> The last thing thats not working is an issue in the feature map normalization. 

Do you mean teacher normalization or anomaly map normalization?

**Comment by alexriedel1** [comment]
> @alexriedel1 Thank you for anomalib, I will comment things which I noticed during reproducing results.
> 
> > The last thing thats not working is an issue in the feature map normalization.
> 
> Do you mean teacher normalization or anomaly map normalization?

I fixed the feature map normalization but cannot reproduce the results. I will have a deeper look into it next week but my guess is the differences in image pre-processing in anomalib and your pre-trained teacher model
